modules/reindex/src/main/java/org/elasticsearch/index/reindex/remote/RemoteRequestBuilders.java

```diff
@@ -170,6 +170,22 @@ static String scrollPath() {
     }
 
     static HttpEntity scrollEntity(String scroll) {
-        return new StringEntity(scroll, ContentType.TEXT_PLAIN);
+        try (XContentBuilder entity = JsonXContent.contentBuilder()) {
+            return new StringEntity(entity.startObject()
+                .field("scroll_id", scroll)
+                .endObject().string(), ContentType.APPLICATION_JSON);
+        } catch (IOException e) {
+            throw new ElasticsearchException("failed to build scroll entity", e);
+        }
+    }
+
+    static HttpEntity clearScrollEntity(String scroll) {
+        try (XContentBuilder entity = JsonXContent.contentBuilder()) {
+            return new StringEntity(entity.startObject()
+                .array("scroll_id", scroll)
+                .endObject().string(), ContentType.APPLICATION_JSON);
+        } catch (IOException e) {
+            throw new ElasticsearchException("failed to build clear scroll entity", e);
+        }
```

Comment:
there is the tiniest difference between them. Clear scroll takes an array and scroll takes a single value